<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mugo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/pics/mugo_128.png</normaloff>:/pics/mugo_128.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="boardTabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuReload">
     <property name="title">
      <string>Reload</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/res/reload.png</normaloff>:/res/reload.png</iconset>
     </property>
     <addaction name="actionEncodingUTF8"/>
     <addaction name="separator"/>
     <addaction name="actionEncodingISO8859_1"/>
     <addaction name="actionEncodingISO8859_15"/>
     <addaction name="actionEncodingWindows_1252"/>
     <addaction name="actionEncodingISO8859_14"/>
     <addaction name="actionEncodingISO8859_7"/>
     <addaction name="actionEncodingWindows_1253"/>
     <addaction name="actionEncodingISO8859_10"/>
     <addaction name="actionEncodingISO8859_3"/>
     <addaction name="separator"/>
     <addaction name="actionEncodingISO8859_4"/>
     <addaction name="actionEncodingISO8859_13"/>
     <addaction name="actionEncodingWindows_1257"/>
     <addaction name="actionEncodingISO8859_2"/>
     <addaction name="actionEncodingWindows_1250"/>
     <addaction name="actionEncodingISO8859_5"/>
     <addaction name="actionEncodingWindows_1251"/>
     <addaction name="actionEncodingKoi8_R"/>
     <addaction name="actionEncodingKoi8_U"/>
     <addaction name="actionEncodingISO8859_16"/>
     <addaction name="separator"/>
     <addaction name="actionEncodingISO8859_11"/>
     <addaction name="actionEncodingISO8859_9"/>
     <addaction name="actionEncodingWindows_1254"/>
     <addaction name="actionEncodingWindows_1258"/>
     <addaction name="separator"/>
     <addaction name="actionEncodingISO8859_6"/>
     <addaction name="actionEncodingWindows_1256"/>
     <addaction name="actionEncodingWindows_1255"/>
     <addaction name="actionEncodingISO8859_8"/>
     <addaction name="separator"/>
     <addaction name="actionEncodingGB2312"/>
     <addaction name="actionEncodingBig5"/>
     <addaction name="separator"/>
     <addaction name="actionEncodingKorean"/>
     <addaction name="separator"/>
     <addaction name="actionEncodingShiftJIS"/>
     <addaction name="actionEncodingJIS"/>
     <addaction name="actionEncodingEucJP"/>
    </widget>
    <widget class="QMenu" name="menuCollection">
     <property name="title">
      <string>Collection</string>
     </property>
     <addaction name="actionCollectionImport"/>
     <addaction name="actionCollectionExtract"/>
    </widget>
    <widget class="QMenu" name="menuRecent_Files">
     <property name="title">
      <string>Recent Files</string>
     </property>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuReload"/>
    <addaction name="actionOpenURL"/>
    <addaction name="separator"/>
    <addaction name="actionCloseTab"/>
    <addaction name="actionCloseAllTabs"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExportBoardAsImage"/>
    <addaction name="actionExportAsciiToClipboard"/>
    <addaction name="separator"/>
    <addaction name="menuCollection"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="menuRecent_Files"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbuot"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <widget class="QMenu" name="menuToolbars">
     <property name="title">
      <string>&amp;Toolbars</string>
     </property>
    </widget>
    <addaction name="menuToolbars"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionDeleteAfterCurrent"/>
    <addaction name="actionDeleteCurrentOnly"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>File Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="commentDockWidget">
   <property name="windowTitle">
    <string>Comment View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="commentDockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="commentWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="branchDockWidget">
   <property name="windowTitle">
    <string>Branch View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="branchDockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QStackedWidget" name="branchStackedWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="undoDockWidget">
   <property name="windowTitle">
    <string>Undo View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="undoDockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QUndoView" name="undoView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="collectionDockWidget">
   <property name="windowTitle">
    <string>Collection View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeView" name="collectionView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="editToolBar">
   <property name="windowTitle">
    <string>Edit Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionDeleteAfterCurrent"/>
  </widget>
  <widget class="QToolBar" name="collectionToolBar">
   <property name="windowTitle">
    <string>Collection Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionCollectionImport"/>
   <addaction name="actionCollectionExtract"/>
   <addaction name="separator"/>
   <addaction name="actionCollectionMoveUp"/>
   <addaction name="actionCollectionMoveDown"/>
   <addaction name="actionCollectionDelete"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>:/res/filenew.png</normaloff>:/res/filenew.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/res/fileopen.png</normaloff>:/res/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionOpenURL">
   <property name="text">
    <string>Open &amp;URL...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:/res/filesave.png</normaloff>:/res/filesave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionCloseTab">
   <property name="text">
    <string>Close Tab</string>
   </property>
  </action>
  <action name="actionCloseAllTabs">
   <property name="text">
    <string>Close All Tabs</string>
   </property>
  </action>
  <action name="actionExportBoardAsImage">
   <property name="text">
    <string>Export Board As Image</string>
   </property>
  </action>
  <action name="actionExportAsciiToClipboard">
   <property name="text">
    <string>Export Ascii To Clipboard</string>
   </property>
  </action>
  <action name="actionCollectionExtract">
   <property name="icon">
    <iconset>
     <normaloff>:/res/collectionextract.png</normaloff>:/res/collectionextract.png</iconset>
   </property>
   <property name="text">
    <string>Extract</string>
   </property>
  </action>
  <action name="actionCollectionImport">
   <property name="icon">
    <iconset>
     <normaloff>:/res/collectionimport.png</normaloff>:/res/collectionimport.png</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionEncodingUTF8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
  <action name="actionEncodingKoi8_R">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cyrillic (KOI8-R)</string>
   </property>
  </action>
  <action name="actionEncodingKoi8_U">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cyrillic/Ukrainian (KOI8-U)</string>
   </property>
  </action>
  <action name="actionEncodingWindows_1250">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Central European (Windows-1250)</string>
   </property>
  </action>
  <action name="actionEncodingWindows_1251">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cyrillic (Windows-1251)</string>
   </property>
  </action>
  <action name="actionEncodingWindows_1252">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Western (Windows-1252)</string>
   </property>
  </action>
  <action name="actionEncodingWindows_1253">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Greek (Windows-1253)</string>
   </property>
  </action>
  <action name="actionEncodingWindows_1254">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Turkish (Windows-1254)</string>
   </property>
  </action>
  <action name="actionEncodingWindows_1255">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hebrew (Windows-1255)</string>
   </property>
  </action>
  <action name="actionEncodingWindows_1256">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Arabic (Windows-1256)</string>
   </property>
  </action>
  <action name="actionEncodingWindows_1257">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Baltic (Windows-1257)</string>
   </property>
  </action>
  <action name="actionEncodingWindows_1258">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vietnamese (Windows-1258)</string>
   </property>
  </action>
  <action name="actionEncodingISO8859_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Western (ISO-8859-1)</string>
   </property>
  </action>
  <action name="actionEncodingISO8859_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Central European (ISO-8859-2)</string>
   </property>
  </action>
  <action name="actionEncodingISO8859_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>South European (ISO-8859-3)</string>
   </property>
  </action>
  <action name="actionEncodingISO8859_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Baltic (ISO-8859-4)</string>
   </property>
  </action>
  <action name="actionEncodingISO8859_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cyrillic (ISO-8859-5)</string>
   </property>
  </action>
  <action name="actionEncodingISO8859_6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Arabic (ISO-8859-6)</string>
   </property>
  </action>
  <action name="actionEncodingISO8859_7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Greek (ISO-8859-7)</string>
   </property>
  </action>
  <action name="actionEncodingISO8859_8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hebrew Visual (ISO-8859-8)</string>
   </property>
  </action>
  <action name="actionEncodingISO8859_9">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Turkish (ISO-8859-9)</string>
   </property>
  </action>
  <action name="actionEncodingISO8859_10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nordic (ISO-8859-10)</string>
   </property>
  </action>
  <action name="actionEncodingISO8859_11">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Thai (ISO-8859-11)</string>
   </property>
  </action>
  <action name="actionEncodingISO8859_13">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Baltic (ISO-8859-13)</string>
   </property>
  </action>
  <action name="actionEncodingISO8859_14">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Celtic (ISO-8859-14)</string>
   </property>
  </action>
  <action name="actionEncodingISO8859_15">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Western (ISO-8859-15)</string>
   </property>
  </action>
  <action name="actionEncodingISO8859_16">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Romanian (ISO-8859-16)</string>
   </property>
  </action>
  <action name="actionEncodingGB2312">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chinese Simplified (GB2312)</string>
   </property>
  </action>
  <action name="actionEncodingBig5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chinese Traditional (Big5)</string>
   </property>
  </action>
  <action name="actionEncodingKorean">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Korean (EUC-KR)</string>
   </property>
  </action>
  <action name="actionEncodingShiftJIS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Japanese (Shift_JIS)</string>
   </property>
  </action>
  <action name="actionEncodingJIS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Japanese (ISO-2022-JP)</string>
   </property>
  </action>
  <action name="actionEncodingEucJP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Japanese (EUC)</string>
   </property>
  </action>
  <action name="actionCopySgfToClipboard">
   <property name="text">
    <string>Copy Sgf to Clipboard</string>
   </property>
  </action>
  <action name="actionCopyCurrentBranchToClipboard">
   <property name="text">
    <string>Copy Current Branch to Clipboard</string>
   </property>
  </action>
  <action name="actionPasteSgfToNewTab">
   <property name="text">
    <string>PasteSgfToNewTab</string>
   </property>
  </action>
  <action name="actionPasteSgfInCollection">
   <property name="text">
    <string>PasteSgfInCollection</string>
   </property>
  </action>
  <action name="actionDeleteAfterCurrent">
   <property name="icon">
    <iconset>
     <normaloff>:/res/remove.png</normaloff>:/res/remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete After Current</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionDeleteCurrentOnly">
   <property name="icon">
    <iconset>
     <normaloff>:/res/removeself.png</normaloff>:/res/removeself.png</iconset>
   </property>
   <property name="text">
    <string>Delete Current Only</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionAbuot">
   <property name="text">
    <string>Abuot...</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionCollectionDelete">
   <property name="icon">
    <iconset>
     <normaloff>:/res/removeself.png</normaloff>:/res/removeself.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionCollectionMoveUp">
   <property name="icon">
    <iconset>
     <normaloff>:/res/collectionup.png</normaloff>:/res/collectionup.png</iconset>
   </property>
   <property name="text">
    <string>Move Up</string>
   </property>
  </action>
  <action name="actionCollectionMoveDown">
   <property name="icon">
    <iconset>
     <normaloff>:/res/collectiondown.png</normaloff>:/res/collectiondown.png</iconset>
   </property>
   <property name="text">
    <string>Move Down</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QUndoView</class>
   <extends>QListView</extends>
   <header>qundoview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
